packages/eslint-config/README.md

```diff
@@ -0,0 +1,46 @@
+## @dotcom-reliability-kit/eslint-config
+
+A prototype linting config, specifically focussed on enhancing code quality and proactively catching errors/bugs before they make it into production. This is not a replacement for the [@financial-times/eslint-config-next](https://github.com/Financial-Times/eslint-config-next), but it can be installed alongside it or as an extension in it's config. This module is part of [FT.com Reliability Kit](https://github.com/Financial-Times/dotcom-reliability-kit#readme).
+
+- [Usage](#usage)
+  - [Static Code Analysis](#static-code-analysis)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Usage
+
+Install @dotcom-reliability-kit/eslint-config as a development dependency:
+
+```
+npm install --save-dev @dotcom-reliability-kit/eslint-tool
```

Comment:
**issue:** just one missed instance of `eslint-tool` here

Reply:
Oh pesky one slipped through. Will do a final search through. Thanks!